### ISSUE TITLE
convergence problem for Cascade Mask RCNN

### ISSUE THREAD

**Issue by TinyGrass** [SwinTransformer/Swin-Transformer-Object-Detection#10]
Thanks for sharing your impressive work.

Recently, I train Swin-based Cascade Mask RCNN and Mask RCNN on my own dataset, including1.5k single-class images. The training process of Mask RCNN is relatively stable. However, the loss of Cascade Mask RCNN only converges to around 1.6 and doesn't decrease anymore. Its validation performance, meanwhile, is lower than the Mask RCNN counterpart. 

They are trained in the same setting, could you give me some suggestions about fine-tuning the Cascade Mask RCNN?

**Comment by HelyModi** [comment]
Hi @TinyGrass , I am facing the same issue for Cascade Mask RCNN. Loss revolves around 1.5-1.7 and does not go lower than that.

**Comment by kentaroy47** [comment]
Cascade trained well in my case.
I used dist_train.sh for training and train.py did not work for cascade.

**Comment by HelyModi** [comment]
> Cascade trained well in my case.
> I used dist_train.sh for training and train.py did not work for cascade.

Hey @kentaroy47, I used dist_train.sh as well, still no improvements.
